Python/bytecodes.c

```diff
@@ -1644,7 +1644,7 @@ dummy_func(
             LOAD_SUPER_ATTR_METHOD,
         };
 
-        inst(LOAD_SUPER_ATTR, (unused/1, global_super, class, self -- res2 if (oparg & 1), res)) {
+        inst(LOAD_SUPER_ATTR, (unused/1, global_super, class, self -- attr, self_or_null if (oparg & 1))) {
```

Comment:
`self_or_null` is unused, so replace with `unused`?

Reply:
Putting `unused` here breaks things. Since this is named it's set to `NULL` in the generated code... if it's called `unused`, it just ignores it and now there's a garbage value there.

Reply:
Oh sneaky. I think you found a hole in the code generator. If you want it to (conditionally) push NULL you should initialize it, not depend on the default initialization. IIRC the default initialization is there only to avoid C compiler warnings.